### ISSUE TITLE
No Validation during training?

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [yhenon/keras-frcnn#172]
In the training code we see there is a validation generator, but actually we do not use it during the training, and we use the total_loss as the metrics.
Should we add validation? It will be a better metrics for saving the best model I guess.

**Comment by rodrigo2019** [comment]
yes it is, the parser create train and test datas, after each epoch you can add a script for a validation, I got better results with validation

**Comment by rodrigo2019** [comment]
![image](https://user-images.githubusercontent.com/8016520/30405756-28c0652a-98c4-11e7-8592-bc137d8b8a71.png)

**Comment by HuangBo-Terraloupe** [comment]
Hello @rodrigo2019,
Can you give more details or share the code how to do the validation, since we are not using model.fit here.

**Comment by rodrigo2019** [comment]
you can download from here #165 , this is my particular train file that I modified from the original

**Comment by thealchemist89** [comment]
@rodrigo2019 
Thanks for sharing your results! Indeed, after epoch 60~100, from your graphs, it suggests the model began overfitting with the divergence.

May I ask how large is your dataset out of curiosity?
